src/models/routes.models.ts

```diff
@@ -0,0 +1,8 @@
+export interface RoutesApp {
+  [key: string]: string;
+}
+
+export const routes: RoutesApp = {
+  HOME: 'Home',
+  ADDCRIPTOCURRENCY: 'add-criptocurrency',
```

Comment:
You should use Screaming Snake Case (ADD_CRYPTO_CURRENCY) here.

Reply:
Okey